src/dataflow-types/src/client.rs

```diff
@@ -71,8 +71,7 @@ pub enum InstanceConfig {
     },
     /// Out-of-process named instance
     Remote {
-        /// The hosts to connect to.
-        hosts: Vec<String>,
+        replicas: Vec<RemoteReplica>,
         /// Logging configuration.
         logging: Option<LoggingConfig>,
```

Comment:
One thing this raises is that the logging stuff isn't going to be well replicated. Peeks and stuff might be ok, but TAIL against any of the introspection sources will be undefined.